### ISSUE TITLE
Spotify priority over game activity

### ISSUE THREAD

**Issue by Duckyz95** [Vencord/plugin-requests#661]
### Plugin Description

A plugin that always prioritises Spotify activity over game activity. There are plugins to quickly disable game activity to acheive this but it's a manual process rather than automatic switching

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by D3SOX** [comment]
Do you mean always showing Spotify activity first? I suppose this would already result in the member list text displaying Spotify instead of a game.

This would be a great addition to my [BetterActivities plugin](https://github.com/D3SOX/vencord-userplugins/tree/master/betterActivities). It already has a setting to prioritize Spotify in the member list.

**Comment by waterbufalowalking** [comment]
He means a plugin to auto-toggle game activity status when Spotify starts playing
bit too niche imo I don't think this will be accepted

**Comment by Duckyz95** [comment]
> He means a plugin to auto-toggle game activity status when Spotify starts playing bit too niche imo I don't think this will be accepted

Yeah that's what I'm after, it is a niche request haha. For now I'll keep using the GameActivityToggle plugin but hopefully someone will accept it

**Comment by D3SOX** [comment]
> He means a plugin to auto-toggle game activity status when Spotify starts playing bit too niche imo I don't think this will be accepted

Ohhhh then I got that completely wrong :facepalm:
